images/miq-app/Dockerfile

```diff
@@ -68,7 +68,9 @@ RUN mkdir -p ${APPLIANCE_ROOT} && \
 ## GIT clone manageiq
 RUN mkdir -p ${APP_ROOT} && \
     ln -vs ${APP_ROOT} /opt/manageiq/manageiq && \
-    curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+    # curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+    curl -L https://github.com/jvlcek/manageiq/tarball/podify_openidc | tar vxz -C ${APP_ROOT} --strip 1
```

Comment:
Looks like this change should go here: 
https://github.com/ManageIQ/manageiq-pods/blob/master/images/manageiq-base/Dockerfile#L74

@carbonin Please confirm.

Reply:
This will not change. When your manageiq changes get merged into master the new build of the pod will include those changes.